### PR TITLE
Move non-core workflows to drawer

### PR DESCRIPTION
**Pull request by pcvera**
Links and one-time settings have been moved to a drawer.

The big hunt name making up the title has been moved into the nav header.

Before
image
After
image
image

### CODE REVIEW

File: cardboard/templates/reactbase.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{% load static %}
```
Comment (pcvera) [inline cardboard/templates/reactbase.html:1]: made a dupe that removes some of the mandatory markup that we can't control from react.

File: hunts/src/HuntViewHeader.js [hunk 2]
```diff
@@ -14,8 +14,6 @@ import OverlayTrigger from "react-bootstrap/OverlayTrigger";
 import Tooltip from "react-bootstrap/Tooltip";
 
 function HuntViewHeader({ hunt }) {
-  const dispatch = useDispatch();
```
Comment (pcvera) [inline hunts/src/HuntViewHeader.js:17]: this wasn't used

File: hunts/src/HuntViewMain.js [hunk 3]
```diff
@@ -69,82 +69,84 @@ export const HuntViewMain = (props) => {
...
-      <Alert
-        dismissible
-        variant={alert.variant}
-        show={alert.show}
-        onClose={() => dispatch(hideAlert())}
-      >
-        {alert.text}
-      </Alert>
-      <HuntViewHeader hunt={hunt} />
+    <>
```
Comment (pcvera) [inline hunts/src/HuntViewMain.js:72]: This file is mostly an indentation change as a result of the fragment I added here

File: hunts/src/DarkModeToggle.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  return (
+    <>
+      <div className="form-check">
+        <input
+          type="checkbox"
+          id="dark-mode-checkbox"
+          className="form-check-input"
+          checked={isDark}
+          onChange={(e) => {
+            console.log(e.target.checked);
```
Comment (asdfryan) [inline hunts/src/DarkModeToggle.js:18]: Remove?

File: hunts/src/DarkModeToggle.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+              localStorage.setItem("cardboard-theme", "bootstrap");
+            } else {
+              // Dark
+              document.body.classList.remove("bootstrap");
+              document.body.classList.add("bootstrap-dark");
+              localStorage.setItem("cardboard-theme", "bootstrap-dark");
+            }
+            setIsDark(e.target.checked);
+          }}
+        ></input>
```
Comment (asdfryan) [inline hunts/src/DarkModeToggle.js:32]: Nit: non-blocking but you can close it via `/>` instead

**Review by asdfryan — APPROVED** [review]

**Pull request merged**